defining variables in repl

```
V 0.1.8
Use Ctrl-D to exit
For now you have to use println() to print values, this will be fixed soon

>>> a := 0
>>> println(a)
panic: vrepl.v:1
undefined: `a`
```

@medvednikov
```
V 0.1.8
Use Ctrl-D to exit
For now you have to use println() to print values, this will be fixed soon

>>> aa:=0
>>> println(aa)
0
>>> println(aa)
panic: vrepl.v:1
redefinition of `aa`
```

--- a/compiler/scanner.v
+++ b/compiler/scanner.v
@@ -213,6 +213,11 @@ fn (s mut Scanner) scan() ScanRes {
 			s.dollar_end = true
 			s.dollar_start = false
 		}
+		if s.pos == 0 && next_char == ` ` {
+			s.pos++
+			//If a single letter name at the start of the file, increment
+			//Otherwise the scanner would be stuck at s.pos = 0
+		}
 		return scan_res(NAME, name)
 	}
 	// number, `.123`

